### PR TITLE
2.0.0

### PR DESCRIPTION
**Pull request by szandrzej**
Incremented deps versions
Adjusted to current Android SDK version
Changed interface to new
Removed unnecessary proxy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,6 +289,7 @@ For example to set the button background color to light green:
 Bug reports and pull requests are welcome through [GitHub Issues](https://github.com/voucherifyio/voucherify-android-sdk/issues).
 
 ## Changelog
+- **2018-04-16** - `2.0.0` - New SDK API
```
Comment (tpindel) [inline README.md]: @szandrzej  Please specify which methods have been adjusted.

File: app/src/main/java/io/voucherify/android/demo/MainActivity.java [hunk 2]
```diff
@@ -32,7 +32,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        voucherifyClient = new VoucherifyAndroidClient.Builder("011240bf-d5fc-4ef1-9e82-11eb68c43bf5", "9e2230c5-71fb-460a-91c6-fbee64707a20")
+        voucherifyClient = new VoucherifyAndroidClient.Builder(
```
Comment (tpindel) [inline app/src/main/java/io/voucherify/android/demo/MainActivity.java:35]: @szandrzej Pleas rollback to the old test account.

File: voucherify-android-sdk/src/main/java/io/voucherify/android/client/model/Tier.java [hunk 3]
```diff
@@ -1,6 +1,10 @@
 package io.voucherify.android.client.model;
 
 public class Tier {
+
+    public static Tier create(String id) {
```
Comment (tpindel) [inline voucherify-android-sdk/src/main/java/io/voucherify/android/client/model/Tier.java]: Please, let's call it `PromotionTier`

File: README.md (hunk 1 above)
Comment (szandrzej) [reply README.md]: done

File: app/src/main/java/io/voucherify/android/demo/MainActivity.java (hunk 2 above)
Comment (szandrzej) [reply app/src/main/java/io/voucherify/android/demo/MainActivity.java:35]: done

File: voucherify-android-sdk/src/main/java/io/voucherify/android/client/model/Tier.java (hunk 3 above)
Comment (szandrzej) [reply voucherify-android-sdk/src/main/java/io/voucherify/android/client/model/Tier.java]: done

**Review by tpindel — APPROVED** [review]

**Pull request merged**